src/Reprocessing_Internal.re

```diff
@@ -294,9 +263,16 @@ let flushGlobalBatch env =>
 
 let maybeFlushBatch ::texture ::el ::vert env =>
   if (
-    env.batch.elementPtr + el >= circularBufferSize ||
-    env.batch.vertexPtr + vert >= circularBufferSize ||
-    env.batch.elementPtr > 0 && env.batch.currTex !== texture
+    env.batch.elementPtr
+    + el
+    >= circularBufferSize
+    || env.batch.vertexPtr
+    + vert
+    >= circularBufferSize
+    || env.batch.elementPtr
+    > 0
+    && env.batch.currTex
+    !== texture
```

Comment:
This still looks bad :(

Reply:
Wasn't this fixed in one of your PRs @IwanKaramazow?

Reply:
This is what it looks like on master
```
let maybeFlushBatch (:texture, :el, :vert, env) =
  if (env.batch.elementPtr
      + el >= circularBufferSize
      || env.batch.vertexPtr
      + vert >= circularBufferSize
      || env.batch.elementPtr > 0
      && env.batch.currTex !== texture) {
    flushGlobalBatch(env)
  };
```
I should probably tune this a bit...

Reply:
@ryanartecona @bsansouci how would you like to see it formatted?

Reply:
I'd vote for something like this.
```reason
env.batch.elementPtr + el >= circularBufferSize
|| env.batch.vertexPtr + vert >= circularBufferSize
|| env.batch.elementPtr > 0
&& env.batch.currTex !== texture
```

I think I'd also be down with introducing some parentheses to disambiguate operator precedence in cases like this which involve long chains of mixed operators.

```reason
(env.batch.elementPtr + el >= circularBufferSize)
|| (env.batch.vertexPtr + vert >= circularBufferSize)
|| (env.batch.elementPtr > 0)
&& (env.batch.currTex !== texture)
```

Though that may be an unpopular opinion or may have already been discussed, I don't know.

Reply:
I think @ryanartecona's format seems good and parens do seems pretty useful as long as we don't get back into the situation of 
```
if (something
    || (
         somethingElse
         && thirdThing
        )
    ) {
...
}
```

So @IwanKaramazow a simple thing right now would be amazing and we can discuss tweaking it on discord / on a more focused issue on the reason repository.

Thanks for commenting!

Reply:
(for reprocessing let's just trust refmt for now)